### ISSUE TITLE
[Bug]: Colour accuracy in OpenTTD with HDR

### ISSUE THREAD

**Issue by SamuXarick** [OpenTTD/OpenTTD#11559]
### Version of OpenTTD

OpenTTD 13.4, Windows 11 Pro 23H2

### Expected result

Accurate colour representation when HDR is enabled in Windows Display Settings

### Actual result

Inaccurate colour representation when HDR is enabled in Windows system.

### Steps to reproduce

I used OBS Studio 30.0.0 to take the screenshots directly from the Preview window. The Source was set to Display Capture with Capture Method set to Automatic, and Force SDR either Off or On.
Each screenshot contains two OpenTTD 13.4 instances side by side, with HW Acceleration Off to the left and On to the right.

1 - (OBS Display Capture - Force SDR Off), (Windows Display Settings - HDR Off)
2 - (OBS Display Capture - Force SDR On), (Windows Display Settings - HDR Off)
3 - (OBS Display Capture - Force SDR Off), (Windows Display Settings - HDR On)
4 - (OBS Display Capture - Force SDR On), (Windows Display Settings - HDR On)

Left - (OpenTTD 13.4 HW Acceleration Off); Right - (OpenTTD 13.4 HW Acceleration On)
![(OBS Display Capture - Force SDR Off), (Windows Display Settings - HDR Off) (OpenTTD - HW Accel Off vs On)](https://github.com/OpenTTD/OpenTTD/assets/43006711/8c7dafad-5d00-45ae-9424-c7a0e4aab8f2)
![(OBS Display Capture - Force SDR On), (Windows Display Settings - HDR Off) (OpenTTD - HW Accel Off vs On)](https://github.com/OpenTTD/OpenTTD/assets/43006711/17e62cb6-df33-404b-8591-22ec0778c96a)
![(OBS Display Capture - Force SDR Off), (Windows Display Settings - HDR On) (OpenTTD - HW Accel Off vs On)](https://github.com/OpenTTD/OpenTTD/assets/43006711/b48ac84b-b30a-4403-9b2d-9ee69e089cfd)
![(OBS Display Capture - Force SDR On), (Windows Display Settings - HDR On) (OpenTTD - HW Accel Off vs On)](https://github.com/OpenTTD/OpenTTD/assets/43006711/92824b30-a159-4cb6-b94f-3f5f7267b79a)

Basically, what I'm asking is to fix screenshot '3' to the right, to have equivalent colour accuracy to that of '2' and '1'.

'4' seems to be an odd case from OBS itself. I intend to use OBS with Force SDR turned Off, ~~as that seems to do just fine with some other games, like Path of Exile~~ (apparently not).

**Comment by PeterN** [comment]
Not an OpenTTD bug.
